config.py

```diff
@@ -0,0 +1,5 @@
+PREFIX = '!'
+TOKEN = ''
+#Todo -- update class dictionary 
```

Comment:
```suggestion
#Todo -- update class dictionary
```
Remove a trailing space. On second thought, could you please remove the trailing spaces from every file? PEP8 recommends that all trailing whitespaces should be removed.

Usually, your editor/IDE will have an option for this.